salt/metalk8s/addons/ui/deployed/ingress.sls

```diff
@@ -90,3 +90,19 @@ spec:
         backend:
           serviceName: metalk8s-ui
           servicePort: 80
+      - path: /config.json
```

Comment:
IIUC, here we want to expose those files to the root, so that can be used by other application. 
Do you think we may have filename conflicts?

Reply:
I agree and I think ultimately we should move those files under the same path of the UI. However right now we have strong dependencies on those files as discussed offline. Also moving the config file under the same path of the UI requires to review how the configuration is injected in the API. Indeed it is this configuration that provides the path on which the UI is deployed, so the UI should be able to get it's configuration no matter the path on which it is deployed (likely by injecting it in index.html when deploying the UI or via another way).